src/main/docs/guide/toc.yml

```diff
@@ -11,4 +11,5 @@ schemasAndPojos: Schemas and POJOs
 schemasAndMetaAnnotations: Schemas and Meta Annotations
 schemasAndGenerics: Schemas and Generics
 mergingSchemas: Merging Schemas
+oauth2: OAuth 2 configuration
```

Comment:
I have renamed this to `Swagger UI - OAuth 2.0 configuration` since it is only related to `swagger-ui`